packages/synapse-react-client/src/components/SynapseTable/SendToCavaticaConfirmationDialog.tsx

```diff
@@ -0,0 +1,165 @@
+  const { data, getLastQueryRequest, onViewSharingSettingsClicked } =
+    useQueryContext()
+  const exportToCavatica = useExportToCavatica(
+    getLastQueryRequest(),
+    data?.queryResult?.queryResults.headers,
+  )
+  const queryRequestCopy = getLastQueryRequest()
+  queryRequestCopy.partMask = SynapseConstants.BUNDLE_MASK_ACTIONS_REQUIRED
+  const { data: asyncJobStatus, isLoading } =
+    useGetQueryResultBundleWithAsyncStatus(queryRequestCopy, {
+      useErrorBoundary: true,
+    })
+
+  const queryResultBundle = asyncJobStatus?.responseBody
+  const actions: ActionRequiredCount[] | undefined =
+    queryResultBundle?.actionsRequired
+  return (
+    <ConfirmationDialog
+      open={showing}
+      title="Send to CAVATICA"
+      content={
+        <>
+          <Box
+            sx={{
+              backgroundColor: 'grey.100',
+              border: '1px solid',
+              borderColor: 'grey.300',
+              marginBottom: '15px',
+              padding: '10px 20px 10px 20px',
+            }}
+          >
+            <Typography variant="body1" sx={{ textAlign: 'center' }}>
+              CAVATICA is a data analysis and sharing platform.
+              {cavaticaHelpURL && (
+                <>
+                  {' '}
+                  Read more about CAVATICA{' '}
+                  <Link href={cavaticaHelpURL} target="_blank">
+                    here
+                  </Link>
+                  .
+                </>
+              )}
+            </Typography>
+          </Box>
+          <Typography
+            variant="body1"
+            sx={{ fontWeight: 700, marginBottom: '10px' }}
+          >
+            You must meet these requirements from CAVATICA to send data:
+          </Typography>
+          <Typography
+            variant="body1"
+            sx={{ marginLeft: '10px', marginBottom: '10px' }}
+          >
+            1. You must be logged in to a CAVATICA account.
+            <br />
+            2. You must connect your CAVATICA account to Synapse.
+          </Typography>
+          <Typography variant="body1">
+            <Link
+              href="https://help.eliteportal.org/help/limited-data-commons#LimitedDataCommons-GainingAccess"
+              target="_blank"
+            >
+              Click here for instructions
+            </Link>
+          </Typography>
+          <Box
+            sx={{
+              backgroundColor: 'grey.100',
+              marginTop: '15px',
+              padding: '10px 20px 10px 20px',
+            }}
+          >
+            <Typography variant="body1">
+              Note that we cannot provide support for CAVATICA. Please contact
+              CAVATICA’s{' '}
+              <Link href="mailto:support@sevenbridges.com "> support</Link> for
+              issues related to the above.
+            </Typography>
+          </Box>
+          {isLoading ? (
+            <SkeletonParagraph numRows={4} />
+          ) : (
+            actions &&
```

Comment:
Won't render anything until we get the `actions` result in the next release of Synapse. This works!